### ISSUE TITLE
Add feature list to README

### ISSUE THREAD

**Issue by dandv** [VioletGiraffe/file-commander#308]
There are [many](https://github.com/IngvarX/Camelot/issues/61) orthodox file managers out there. I'm looking for one that supports [launching external tools to compare files](https://github.com/mucommander/mucommander/issues/928).

Would be informative to list the application features in the README, in addition to the nice screenshots 🙏

**Comment by VioletGiraffe** [comment]
I agree, I did want to compile a list of features, especially since some are not discoverable from the menus and other GUI elements.

There is no support for external tools that receive two files (rather than one), but it would be pretty simple to add. There is a built-in file comparison tool, but it only says whether or not the files are equal.

**Comment by xparq** [comment]
If you get to it, you might add a section about the important differences from TC (apart from being cross-platform), if any. (That could be efficient also for you, allowing to spare listing all the bog standard features TC also have.) I don't mean wasting effort on a detailed "formal" comparison, just an overview.
